### PR TITLE
Add cmp and dev for I²C based SGP sensors

### PR DESCRIPTION
**Pull request by dbrgn**
Like #1 but for SGP.

I'm using the generic IC symbol here.

### CODE REVIEW

**Comment by dbrgn** [comment]
This is actually wrong, we need the 5-pin symbol :stuck_out_tongue:

**Comment by rnestler** [comment]
There are two branches in this repo, one named `sgp30` and the other named `sgp`. Are both meant to be merged?

**Comment by rnestler** [comment]
Also the PR should be rebased, since it is quite behind :slightly_smiling_face:

**Comment by dbrgn** [comment]
@rnestler branch is now updated with the fixed component, as well as with two devices for `SGP30` and `SGPC3`. Maybe you can take a look now?

File: dev/41c899fa-89b1-4d23-93a1-e03f6f5e88eb/device.lp [hunk 1]
```diff
@@ -0,0 +1,19 @@
+(librepcb_device 41c899fa-89b1-4d23-93a1-e03f6f5e88eb
+ (name "SGP30")
+ (description "Gas sensor for indoor air quality applications.")
```
Comment (rnestler) [inline dev/41c899fa-89b1-4d23-93a1-e03f6f5e88eb/device.lp]: Do you want to add a link to the datasheet in the description? https://www.sensirion.com/file/datasheet_sgp30/.

File: dev/41c899fa-89b1-4d23-93a1-e03f6f5e88eb/device.lp [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ (keywords "gas,voc,tvoc,co2eq,sensor")
```
Comment (rnestler) [inline dev/41c899fa-89b1-4d23-93a1-e03f6f5e88eb/device.lp]: Maybe add 'sensirion' as a keyword?

File: dev/41c899fa-89b1-4d23-93a1-e03f6f5e88eb/device.lp [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ (author "Danilo Bargen")
+ (version "0.1")
+ (created 2019-11-08T23:24:14Z)
+ (deprecated false)
+ (category 8c508cce-7904-44b9-a300-d9cc6eb44a6d)
+ (component 2f578e3d-76e4-4d6b-b635-a3041466a377)
+ (package 888ec1db-8bc5-4b66-ac62-e8f569e02c63)
```
Comment (rnestler) [inline dev/41c899fa-89b1-4d23-93a1-e03f6f5e88eb/device.lp]: The package footprint looks quite different form the one in the datasheet:
sgp-footprint
Is this intended?

File: dev/f5dc3e3e-ff41-45c9-93f2-86052e0178c5/device.lp [hunk 4]
```diff
@@ -0,0 +1,19 @@
+(librepcb_device f5dc3e3e-ff41-45c9-93f2-86052e0178c5
+ (name "SGPC3")
+ (description "Ultra-low power gas sensor for indoor air quality applications.")
```
Comment (rnestler) [inline dev/f5dc3e3e-ff41-45c9-93f2-86052e0178c5/device.lp]: Same here: Do you want to add a link to the datasheet in the description? https://www.sensirion.com/file/datasheet_sgpc3/.

**Review by rnestler — COMMENTED** [review]
I'm not sure if the generic DFN package works optimally for the SGPs. @ubruhin what do you think?

File: dev/41c899fa-89b1-4d23-93a1-e03f6f5e88eb/device.lp (hunk 2 above)
Comment (dbrgn) [reply dev/41c899fa-89b1-4d23-93a1-e03f6f5e88eb/device.lp]: Not sure if we should add the manufacturer of every component to the keywords, but it doesn't hurt.

**Comment by ubruhin** [comment]
> I'm not sure if the generic DFN package works optimally for the SGPs. @ubruhin what do you think?

Yes we should use a SGP specific package anyway because of the very special sensing "hole" at the top...

**Comment by dbrgn** [comment]
Ah, good point. We discussed that some time ago, but I forgot to add a TODO note to this PR :slightly_smiling_face:

File: dev/41c899fa-89b1-4d23-93a1-e03f6f5e88eb/device.lp (hunk 3 above)
Comment (dbrgn) [reply dev/41c899fa-89b1-4d23-93a1-e03f6f5e88eb/device.lp]: Updated.

**Comment by dbrgn** [comment]
Devices updated with custom package. The package was generated with https://github.com/LibrePCB/librepcb-parts-generator/pull/70.

**Review by ubruhin — APPROVED** [review]

**Pull request merged**